lms/static/sass/views/_account-settings.scss

```diff
@@ -36,45 +39,269 @@
+            padding-top: ($baseline*2);
         }
 
         .header-subtitle {
             color: $gray-l2;
         }
+
+        .account-nav {
+        margin: ($baseline/2) 0px;
+        padding: 0px;
+        list-style: none;
+        @include float(left);
+        .account-nav-item{
+            display: flex;
+            margin: 0px;
+            text-transform: none;
+            @include float(left);
+            justify-content: center;
+            .account-nav-link{
+                font-size: em(16);
+                color: #7a7979;
+                padding: 5px 25px 23px;
+                display: inline-block;
+            }
+            .active{
+                border-bottom: 4px solid $courseware-border-bottom-color !important;
+            }
+        }
+        li,
+        div {
+            display: inline-block;
+            margin: 0 $baseline+1 0 0;
+            font-size: em(14);
+            font-weight: 600;
+            text-transform: uppercase;
+            letter-spacing: 0 !important;
+
+            &:last-child {
+                margin-right: 0;
+            }
+
+            a {
+                display:block;
+                padding: ($baseline/4);
+                color: $link-color;
+                font-weight: 600;
+
+                &:hover, &:focus, &:active {
+                    text-decoration: none;
+                    color: $link-color;
+                }
+            }
+
+            &.active {
+
+                a {
+                    text-decoration: none;
+                    color: $link-color;
+                }
+            }
+        }
+      }
     }
 }
 
 // +Settings Section
 .account-settings-sections {
+    padding-bottom: ($baseline*5);
 
     .section-header {
         @extend %t-title6;
         @extend %t-strong;
-        padding-bottom: ($baseline/2);
-        border-bottom: 1px solid $gray-l4;
+        padding-top: ($baseline/2)*3;
+        color: #5a5959;
     }
 
     .section {
         background-color: $white;
-        padding: $baseline;
         margin-top: $baseline;
-        border: 1px solid $gray-l4;
-        box-shadow: 0 0 1px 1px $shadow-l2;
-        border-radius: 5px;
-    }
+        border-bottom: 4px solid #f5f5f5;
 
-    a span {
-        color: $link-color;
-    }
+        .account-settings-header-subtitle {
+            font-size: em(18);
+            color: #7a7979;
+            padding-top: 20px;
+            padding-bottom: 10px;
+        }
+
+        .account-settings-section-body {
+
+            .u-field {
+                border-bottom: 2px solid #f5f5f5;
+
+                .field {
+                    width: 30%;
+                    vertical-align: top;
+                    display: inline-block;
+                    position: relative;
+
+                    select {
+                        -webkit-appearance: none;
+                        -moz-appearance: none;
+                        appearance: none;
```

Comment:
Please use the appearance mixin already defined https://github.com/edx/edx-platform/blob/76b8e2e89761c30ac7aa2fa5e8aca4dc35d9002b/common/static/sass/bourbon/css3/_appearance.scss#L1